app/javascript/article-form/components/ClipboardButton.jsx

```diff
@@ -1,30 +1,14 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
-import { Button } from '@crayons';
+import { ButtonNew as Button } from '@crayons';
```

Comment:
This `ButtonNew as Button` is temporary until we remove old Button component from Crayons. But then it should be as simple as using _Find all and replace_ command.